sdk/storage/storage-blob/samples/storage-blob-resize-function/Thumbnail/index.js

```diff
@@ -0,0 +1,44 @@
+const Jimp = require('jimp');
+const stream = require('stream');
+const {
+    BlockBlobClient
+} = require("@azure/storage-blob");
+
+const ONE_MEGABYTE = 1024 * 1024;
+const ONE_MINUTE = 60 * 1000;
+const uploadOptions = { bufferSize: 4 * ONE_MEGABYTE, maxBuffers: 20 };
+
+
+const containerName = process.env.BLOB_CONTAINER_NAME;
```

Comment:
Maybe better to invoke dotenv like:

```
// Load the .env file if it exists
require("dotenv").config();
```

Reply:
I try to get variable with the package `dotenv`, but get nothing. It will be work by setting in the azure function.
Go to Azure portal and find your Azure Function,  select `Configuration` under the `Settings` tab, add new application setting for the variables required by the sample.